src/mapfile2js/parse/checkComment.ts

```diff
@@ -1,12 +1,18 @@
-const regExpHexColor = new RegExp('["\']#[0-9a-f]{6}["\']|["\']#[0-9a-f]{3}["\']', 'gi');
+const regExpHexColor = new RegExp('["\']#[0-9a-f]{6,8}["\']|["\']#[0-9a-f]{3}["\']', 'gi');
```

Comment:
The regex isn't as exact as it could be, that would be to big of a change, I guess.